code/test/README.md

```diff
@@ -0,0 +1,113 @@
+# Contributors Testing Guide
```

Comment:
Overall the framework is nice, but some structure might need some revision - some test files are getting too big and there is too much repeated code. We probably don't need too much repeated basic check after each test for specialized operations.  Testing the logics and expected use cases of each operation is much more important than coverage of basic invocation. See innerCore.js and spendLimit.js tests for example. These tests later guided the design of the UI and some subtleties in how effectiveTime should be set and how to find correct nodes in merkle trees. 

Also  when functionalities evolve or change, it may force too much code change.